opencensus/proto/agent/trace/v1/trace_service.proto

```diff
@@ -32,16 +32,16 @@ option go_package = "github.com/census-instrumentation/opencensus-proto/gen-go/a
 service TraceService {
   // After initialization, this RPC must be kept alive for the
   // entire life of the application. The agent pushes configs
-  // down to applications via a stream of requests.
-  rpc Config(stream ConfigTraceServiceResponse) returns (stream ConfigTraceServiceRequest) {}
+  // down to applications via a stream.
+  rpc Config(stream CurrentLibraryConfig) returns (stream LibraryConfigUpdate) {}
```

Comment:
How about updatedLibraryConfig or newLibraryConfig instead of LibraryConfigUpdate?

Reply:
Among the 3 names "UpdatedLibraryConfig" sounds the most appropriate to me, though I don't have a strong opinion. WDYT @odeke-em @bogdandrutu ?

Reply:
+1 for UpdatedLibraryConfig

Reply:
Thanks, looks like we have a consensus here so I renamed it to "UpdatedLibraryConfig". Also re-generated the gen-go files.

Reply:
Nice suggestion with UpdatedLibraryConfig @rghetia, SGTM!